openapi/social/social.yaml

```diff
@@ -0,0 +1,176 @@
+          in: query
+          name: 'page[cursor]'
+          description: The cursor stores all your filters and current location in the list to allow paging over the results in smaller batches. The value will be provided in the response links.
+        - schema:
+            type: integer
+            example: 20
+            default: 25
+          in: query
+          name: 'page[limit]'
+          description: The maximum number of Social Profiles you would like to return.
+      tags:
+        - Social Profiles
+      x-lifecycle:
+        Status: Trusted Tester
+    options:
+      summary: ''
+      operationId: options-socialProfiles
+      description: 'Used solely for [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) the OPTIONS request returns the list of possible HTTP methods and other headers that browsers use to protect user''s security. You should not call this operation directly. '
+      responses:
+        '204':
+          description: No Content
+      tags:
+        - Options
+        - Social Profiles
+components:
+  securitySchemes:
+    JWT:
+      type: http
+      scheme: bearer
+      bearerFormat: JWT
+    OAuth2:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://sso-api-demo.apigateway.co/oauth2/auth'
+          tokenUrl: 'https://sso-api-demo.apigateway.co/oauth2/token'
+          scopes:
+            business: Read-write access to business details
+            social: Read-write access to Social Marketing APIs
+  schemas:
+    socialProfiles:
+      title: Social Profiles
+      type: object
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        An entity that contains the information of a connected account or page with a social network.
+      x-tags:
+        - Social Profiles
+      properties:
+        type:
+          type: string
+          default: socialProfiles
+        id:
+          type: string
+          example: FBU-117641035298609261021
+          description: The unique id for this Social Profile.
+        attributes:
+          type: object
+          properties:
+            name:
+              type: string
+              example: John's Delivery
+              description: The name of this profile.
+            avatarUrl:
+              type: string
+              description: The link to an url for the avatar image of this profile.
+            socialNetwork:
+              type: string
+              description: A string enum representing which social network this Social Profile is on.
+              enum:
+                - facebook
+                - twitter
+                - gmb
+                - linkedinCompany
+                - linkedinUser
+                - instagram
+            status:
+              type: string
+              description: A string enum that describes the current status of this Social Profile.
```

Comment:
Could you explain what the status field is without using the word status? Providing examples of when a social profile would be in the disabled or active status may help.